websites/A/Aqua/dist/metadata.json

```diff
@@ -1,21 +1,20 @@
 {
   "author": {
-    "name": "Jack Hansen",
-    "id": "140590672657776641"
+    "name": "Ben",
+    "id": "781285747058933790"
   },
-  "service": "Dual",
+  "service": "Aqua",
   "description": {
-    "en": "Let Dual power your ears with the latest and greatest music from around the world."
+    "en": "Aqua radio brings you the greatest hits from around the world"
   },
-  "url": "dual.pw",
-  "version": "1.0.7",
-  "logo": "https://i.imgur.com/JQ1t9N5.png",
-  "thumbnail": "https://i.imgur.com/zIFABSa.png",
-  "color": "#D08CCF",
+  "url": "itsaqua.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/dBqeZ3q.png",
+  "thumbnail": "https://i.imgur.com/CiE0uNv.png",
+  "color": "#2980b9",
```

Comment:
why is that the thumbnail?

Reply:
was under the impression the thumbnail is the background on the site? like the little background

Reply:
a promotional banner, or a screenshot of the website homepage.

Reply:
ahhh okay - this is a super blurred screenshot of one of our pages, would you like me to submit it unblurred?

Reply:
Yes, it only needs blurring if its NSFW